src/lib/utilities/deleteAllChats.ts

```diff
@@ -0,0 +1,77 @@
+export const deleteAllChats = (): Error | void => {
+    try {
+        const chatHistory = document.querySelector(
+            "div.group\\/sidebar> div.flex.flex-col.gap-2.text-token-text-primary.text-sm.false.mt-5.pb-2",
+        );
+
+        if (!chatHistory) throw new Error("No chat history found");
+
+        const profileButton = document.querySelector(
+            '[aria-label="Open Profile Menu"]',
+        ) as HTMLButtonElement;
+
+        if (!profileButton) throw new Error("Profile button not found!");
+
+        profileButton.dispatchEvent(
```

Comment:
add inline comment here explaining that the pointerdown and pointerup events need to be manually set since click() and mouseclick events don't seem to work here. also, consider making this a util in case we want to use this new pointer simulation in other places in the future.